setup.py

```diff
@@ -53,7 +53,6 @@
         "pyarrow >= 1.0.0, < 2.0dev",
     ],
     "tqdm": ["tqdm >= 4.7.4, <5.0.0dev"],
-    "fastparquet": ["fastparquet", "python-snappy", "llvmlite>=0.34.0"],
```

Comment:
I'd like to see us add "pyarrow" to the "pandas" extras now, since it's needed for both uploads and downloads to dataframe.

We can maybe refactor the `pyarrow >=1.0.0,<2.0dev` string into a variable since it's going to appear 3 times in setup.py now too